### ISSUE TITLE
Issue with requesting Performance Storage - IOPS is hardcoded

### ISSUE THREAD

**Issue by surajsub** [IBM-Cloud/terraform-provider-ibm#287]
### Terraform Version
0.11.3

### Affected Resource(s)
Please list the resources as a list, for example:

- ibm_storage_block

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```
## Added the provider version

variable "datacenter" {
	description = "Target vSphere datacenter for Virtual Machine creation"
}


variable "iops" {
	description = "IOPS for the storage"
	
	
	}


variable "size" {
	description = "Size of the storage"
	
	
	}


provider ibm {
version="0.8"
} 

variable "ipToConnect" {
description="Host to connect"
type="list"
}


variable "os_format_type"
{
description="OS Format Type to Provide"
}





resource "ibm_storage_block" "cmas" {
        type = "Performance"
        datacenter = "${var.datacenter}"
        capacity = "${var.size}"
        iops = "${var.iops}"
        os_format_type = "${var.os_format_type}"
        allowed_ip_addresses =["${var.ipToConnect}"]
        
}

output "hostname" {
  value = "${ibm_storage_block.cmas.hostname}"
}
output "storageId" {
value = "${ibm_storage_block.cmas.id}"
}
output "volumename" {
  value = "${ibm_storage_block.cmas.volumename}"
}
output "allowed_host_info" {
  value = "${ibm_storage_block.cmas.allowed_host_info}"
}

```

### Panic Output
N/A

### Expected Behavior
Performance storage should have been created.
### Actual Behavior
The provider seems to be searching for a iops with a different value and ignoring the input

Output 

Tue Jun 19 2018 17:55:23 GMT+0000 (UTC) Running terraform init ... 
Initializing provider plugins...
Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.
If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
Tue Jun 19 2018 17:55:23 GMT+0000 (UTC) Running terraform apply ... 
ibm_storage_block.cmas: Creating...
  allowed_hardware_ids.#:          "" => ""
  allowed_hardware_info.#:         "" => ""
  allowed_host_info.#:             "" => ""
  allowed_ip_addresses.#:          "" => "1"
  allowed_ip_addresses.1293654865: "" => "10.63.43.104"
  allowed_virtual_guest_ids.#:     "" => ""
  allowed_virtual_guest_info.#:    "" => ""
  capacity:                        "" => "12000"
  datacenter:                      "" => "syd04"
  hostname:                        "" => ""
  hourly_billing:                  "" => "false"
  iops:                            "" => "5800"
  os_format_type:                  "" => "Linux"
  type:                            "" => "Performance"
  volumename:                      "" => ""

Error: Error applying plan:
1 error(s) occurred:
* ibm_storage_block.cmas: 1 error(s) occurred:
* ibm_storage_block.cmas: Error while creating storage:No product items matching with keyName 1000_6000_IOPS and categoryCode performance_storage_iops could be found
Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.

**Comment by hkantare** [comment]
We are not able to reproduce with latest terraform provider v0.10.0 . We made a fix in v0.10.0 (https://github.com/IBM-Cloud/terraform-provider-ibm/pull/250) Can you please try with latest provider.
```
ibm_storage_block.bs_performance:
  id = 45451469
  allowed_hardware_ids.# = 0
  allowed_hardware_info.# = 0
  allowed_host_info.# = 0
  allowed_ip_addresses.# = 0
  allowed_virtual_guest_ids.# = 0
  allowed_virtual_guest_info.# = 0
  capacity = 12000
  datacenter = syd04
  hostname = 161.26.126.18
  hourly_billing = true
  iops = 5800
  notes = performance notes
  os_format_type = Linux
  type = Performance
  volumename = IBM02SL1204953-5
ibm_storage_block.bs_performance_monthly:
  id = 45451647
  allowed_hardware_ids.# = 0
  allowed_hardware_info.# = 0
  allowed_host_info.# = 0
  allowed_ip_addresses.# = 0
  allowed_virtual_guest_ids.# = 0
  allowed_virtual_guest_info.# = 0
  capacity = 12000
  datacenter = syd04
  hostname = 161.26.126.16
  hourly_billing = false
  iops = 5800
  notes = performance notes
  os_format_type = Linux
  type = Performance
  volumename = IBM02SL1204953-6
```

**Comment by surajsub** [comment]
After the update to 0.10.0 - this worked. Thanks . Appreciate the quick response.
